packages/swatch/src/Swatch.ts

```diff
@@ -229,6 +229,11 @@ export class Swatch extends SizedMixin(Focusable, {
                 this.removeAttribute('aria-label');
             }
         }
+        if (changes.has('mixedValue')) {
+            if (this.mixedValue) {
+                this.setAttribute('aria-checked', 'mixed');
+            }
+        }
```

Comment:
Hey Rajdeep! Just following up on our conversation earlier today. 

To add a label to the `Swatch` that matches the `color` attribute, you can add the following in the Swatch code's willUpdate function: 

```        
if (changes.has('label')) {
            // if there is a colour set,
            if (this.color !== '') { 
                this.setAttribute('aria-label', this.color); // set the aria-label to be this.color
           // otherwise, if the label is different from the colour and is a length longer than 0,
            } else if (this.label !== this.color && this.label.length) { 
                // override the label with this.label
                this.setAttribute('aria-label', this.label); 
            }
             else {
                this.removeAttribute('aria-label');
            }
        }
```

This obviously doesn't handle situations where our value would be 'mixed' or 'nothing' (since `color=''` in that situation), but it's better than no label at all. Hopefully, we can get some guidance on @dineshy87 's side as to what kind of aria-label would be best for those situations, too. 😄 Depending on how many different types of labels we need, we may want to abstract this out of `willUpdate()`.

Reply:
@najikahalsema Thanks!!

<img width="1792" alt="Screenshot 2023-06-23 at 3 41 23 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/c2d6c550-016f-4f06-90af-70d5ffebbfcf">